.github/shared/src/swagger.js

```diff
@@ -118,6 +129,56 @@ export class Swagger {
     return filtered;
   }
 
+  /**
+   * @returns {Promise<Map<string, Operation>>}
+   */
+  async getOperations() {
+    if (!this.#operations) {
+      this.#operations = new Map();
+      const content = await readFile(this.#path, "utf8");
```

Comment:
Assuming the file content is needed in multiple APIs (getOperations and getVersion), we should ensure the file is only read once.  Easiest way, when either value is needed, compute and cache both.

Reply:
File content will only be read in this function. I removed the content access in getVersion function.